html/index.py

```diff
@@ -834,10 +834,15 @@
                     errors.push("Address is required!");
                     $('#send_address').parent().addClass('has-error');
                 }
-                else if(!((address.substr(0, 4) == "Sumo" && address.length == 99) ||
-                    (address.substr(0, 4) == "Sumi"  && address.length == 110) ||
-                    (address.substr(0, 4) == "Subo"  && address.length == 98)))
-                {
+                else if(!(address.substr(0, 4) == "cash" && address.length == 98)) {
+                    if (!(address.length == 98)) {
+                        errors.push("Address is not valid! [DEBUG MESSAGE] Length may need to be set to 99");
+                    } else {
+                        errors.push("Address is not valid!");
+                    }
+                    $('#send_address').parent().addClass('has-error');
+                } else if (!((address.substr(0, 4) == "casi"  && address.length == 110) ||
+                (address.substr(0, 4) == "Subo"  && address.length == 98))) {

```

Comment:
Support for Subo may cause confusion in the future if a user or exchange accidentally sends Sumokoin to an Ombre wallet. I would suggest coming up with a brand new address prefix if a third one is needed.